src/utils/dateUtils.ts

```diff
@@ -0,0 +1,10 @@
+export const isAfter = (dateString1: string | undefined, dateString2: string | undefined): boolean => {
+  if (dateString1 === undefined || dateString2 === undefined) {
+    return false;
```

Comment:
I wonder if returning false here is correct. It would imply that dateString1 is on or before dateString2, and I wonder whether null or undefined is the correct return value.. Of course, where you used it it won't make a difference, but just worrying over the semantics if this is reused elsewhere.

Reply:
Thanks for the feedback. The current behavior should be ok as long as the contract is documented. I can at least do that. There is little value in returning undefined etc. Alternatively I could make the input required.

Reply:
I could look at the various combinations of undefined, that might help. Let me do that.

Reply:
Nice fix!